test/NinjaTools.FluentMockServer.Tests/Serialization/ExpectationSerializationTests.cs

```diff
@@ -30,24 +76,27 @@ public void SerializeJObject_Should_Use_SerializeJObject_Of_Properties()
                 .WithPath("")
                 .WithMethod(HttpMethod.Post)
                 .WithContent(body => body.WithoutXmlContent("<xml here>"));
-         
-            var expectation = FluentExpectationBuilder.Create(
-                httpRequest: builder.Build(),
-                httpResponse: HttpResponse.Create(headers: new Dictionary<string, string[]>
+
+            var expectation = FluentExpectationBuilder.Create(builder.Build(), new HttpResponse
+                {
+                    Headers = new Dictionary<string, string[]>
                     {
                         {"Content-Type", new[] {"xml"}}
                     },
-                    body: new JValue("some xml response"),
-                    delay: new Delay(TimeUnit.Milliseconds, 50),
-                    statusCode: 200)
+                    Body = new JValue("some xml response"),
+                    Delay = new Delay(TimeUnit.Milliseconds, 50),
+                    StatusCode = 200
+                }
             );
             
             
             // Act
-            var jo = expectation.AsJObject();
+            var jo = Serializer.SerializeJObject(expectation);
             var json = Serializer.Serialize(expectation);
-
-            Logger.LogInformation("JSON", json);
+            
+            Dump(jo, "JObject");
+            Dump(json, "Json");
+            //Logger.LogInformation("JSON", json);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/git_36/NinjaTools.FluentMockServer/pullRequest?prid=4885942)